### PR TITLE
increase parallelization of the update method

### PR DESCRIPTION
**Pull request by brastak**
For #11

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#14](https://github.com/artipie/go-adapter/issues/14) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/597e70b1-5231-4fa4-baae-9901dbd88d5d); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/goproxy/Goproxy.java [hunk 1]
```diff
@@ -111,27 +111,31 @@ public Completable update(final String repo, final String version) {
     private Completable actualUpdate(final String repo, final String version) throws IOException {
         final String[] parts = repo.split("/", 2);
         final String lkey = String.format("%s/@v/list", repo);
-        return this.loadGoModFile(parts)
-            .flatMapCompletable(
+        return Completable.concatArray(
+            this.loadGoModFile(
+                parts
```
Comment (carlosmiranda) [inline src/main/java/com/artipie/goproxy/Goproxy.java]: @brastak minor formatting comment - let's inline `parts` parameter with `loadGoModFile` call, similar to previous formatting.

File: src/main/java/com/artipie/goproxy/Goproxy.java [hunk 2]
```diff
@@ -111,27 +111,31 @@ public Completable update(final String repo, final String version) {
...
             ).flatMapCompletable(
                 zip -> this.storage.save(
                     new Key.From(String.format("%s/@v/v%s.zip", repo, version)),
                     new Content.From(new RxFile(zip, this.vertx.fileSystem()).flow())
                 ).andThen(Completable.fromAction(() -> Files.delete(zip)))
-            ).andThen(generateVersionJson(version))
-            .flatMapCompletable(
+            ),
+            generateVersionJson(
+                version
```
Comment (carlosmiranda) [inline src/main/java/com/artipie/goproxy/Goproxy.java]: @brastak same here, let's inline `version` param with `generateVersionJson`

File: src/main/java/com/artipie/goproxy/Goproxy.java (hunk 1 above)
Comment (brastak) [reply src/main/java/com/artipie/goproxy/Goproxy.java]: done

File: src/main/java/com/artipie/goproxy/Goproxy.java (hunk 2 above)
Comment (brastak) [reply src/main/java/com/artipie/goproxy/Goproxy.java]: done

**Comment by brastak** [comment]
@carlosmiranda Please, check update

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by carlosmiranda** [comment]
@brastak need to merge with master.

**Pull request merged**